### DISCUSSION TITLE
Conflicting information in documentation about Test Base package creation.

### DISCUSSION THREAD

**Discussion by lordarokh** [microsoft/testbase#39] [Announcements]
Hi There,

While reading the online documentation for Test Base, I came over the following conflicting information:

The document which explains "[Building a Package](https://learn.microsoft.com/en-us/microsoft-365/test-base/buildpackage?view=o365-worldwide)" states that the folder "LOGS" should be created in **root** of the application package :

> _These folders are used:_
_App\bin: save the application and dependency binaries._
_App\scripts: save scripts to install, launch, close and uninstall your application._
_**App\logs**: scripts should output logs to this folder, then you can download and analyze logs after test is finished.__
 
However, the documentation "[Test package guidelines](https://learn.microsoft.com/en-us/microsoft-365/test-base/contentguideline?view=o365-worldwide)" states that the "LOGS" folder needs to be under each subsequent action script folder :

> _For example, the installation script (which may be located in the App/scripts/install directory) can output its logs to: logs/install.log, such that the final log will be at: _**Apps/scripts/install/logs/install.log**__

So any idea which one is correct?

**Comment by Tinacyt** [comment]
Hi there,

That's a good catch! 😉

In fact, when building a Test Base package, the position/sequence of the 'LOGS' folder is very flexible, as long as all the 'logs' scripts are under the 'LOGS' folder.

The two documentations mentioned in your question are just showing some examples for your reference, meaning that the 'LOGS' folder position does not need to be exactly the same as the examples.

We will add more explanatory notes to those documentations to avoid any confusion on that. Thank you for bringing this up!

Regards,
Tina
